modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -28,35 +28,44 @@ When using the {product-title} (ROSA) CLI, `rosa`, to create a cluster, you can
 //REGION="<region>"
 //----
 
-. You can create your {hcp-title} cluster with one of the following commands.
+. The following command creates your {hcp-title} cluster.
 +
 [NOTE]
 ====
 When creating a {hcp-title} cluster, the default machine Classless Inter-Domain Routing (CIDR) is `10.0.0.0/16`. If this does not correspond to the CIDR range for your VPC subnets, add `--machine-cidr <address_block>` to the following commands. To learn more about the default CIDR ranges for {product-title}, see xref:../networking/cidr-range-definitions.adoc#cidr-range-definitions[CIDR range definitions].
 ====
 +
-** Create a cluster with a single, initial machine pool, publicly available API, and publicly available Ingress by running the following command:
-+
+--
 [source,terminal]
 ----
-$ rosa create cluster --cluster-name=<cluster_name> \
-    --sts --mode=auto --hosted-cp --operator-roles-prefix <operator-role-prefix> \
-    --oidc-config-id <ID-of-OIDC-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id>
+$ rosa create cluster --cluster-name=<cluster_name> \ <.>
+    --sts --mode=auto --hosted-cp [--private] \ <.> 
+    --operator-roles-prefix <operator-role-prefix> \ <.>
+    --oidc-config-id <id-of-oidc-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id>
```

Comment:
Is this correct? I see a --private flag but the --subnet-ids have public-subnet-id

Reply:
How about wrap `--subnet-ids=<public-subnet-id>,<private-subnet-id>` into a new line?

Reply:
@bchandra-ocp @xueli181114 
How about this?
![image](https://github.com/openshift/openshift-docs/assets/16167833/48c1fc98-0f78-43b2-ae1d-7eb4a42a9342)